src/Mixin.php

```diff
@@ -0,0 +1,539 @@
+<?php
+
+/**
+ * Automatically generated
```

Comment:
Better comment/note needed here, specifically about what auto-generated it, and why it is here

Reply:
Can we please brainstorm it

```php
/**
 * Automatically generated
 * 
 * Provides psalm assertions for `nullOr` and `all` methods  
 */
```

Reply:
"provides type inference and auto-completion for magic static methods of `Assert`"